src/psyclone/parse/algorithm.py

```diff
@@ -256,7 +258,8 @@ def create_invoke_call(self, statement):
                         format(str(statement), self._alg_filename))
                 invoke_label = self.check_invoke_label(argument)
 
-            elif isinstance(argument, (Data_Ref, Part_Ref)):
+            elif isinstance(
```

Comment:
Please could you add/update `__all__` at the bottom of this file.

Reply:
Added.